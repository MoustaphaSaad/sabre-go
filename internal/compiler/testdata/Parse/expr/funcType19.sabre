func() (func(), func(x, y))