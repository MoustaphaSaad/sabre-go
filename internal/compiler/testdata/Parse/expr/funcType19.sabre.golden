(FuncType
  (FuncType-Results
    (FuncType)
    (FuncType
      (FuncType-Parameters
        (NamedType IDENTIFIER(x))
        (NamedType IDENTIFIER(y))
      )
    )
  )
)